src/tike/communicators/comm.py

```diff
@@ -48,31 +48,39 @@ def __exit__(self, type, value, traceback):
             self.mpi.__exit__(type, value, traceback)
         self.pool.__exit__(type, value, traceback)
 
-    def reduce(self, x, dest, **kwargs):
+    def reduce(self, x, dest, s=1, **kwargs):
```

Comment:
What is `s`? For each function that you changed the API, please update the docstrings.

Reply:
Added.